idea/src/org/jetbrains/kotlin/idea/highlighter/KotlinSuspendCallLineMarkerProvider.kt

```diff
@@ -108,10 +109,11 @@ fun KtExpression.hasSuspendCalls(bindingContext: BindingContext = analyze(BodyRe
             }
         }
         else -> {
-            val resolvedCall = getResolvedCall(bindingContext)
-            if ((resolvedCall?.resultingDescriptor as? FunctionDescriptor)?.isSuspend == true) true
+            val target = (this as? KtParenthesizedExpression)?.getStrictParentOfType<KtCallExpression>() ?: this
```

Comment:
This is a very hacky way to solve the problem. The root of problem here is the fact that the considered call expression is not a valid candidate (see check at line 88), so I believe this place should be fixed instead.